src/AccessibilityInsights.Core/Types/TypeBase.cs

```diff
@@ -115,5 +115,10 @@ public bool Exists(int id)
         {
             return this.Dic.ContainsKey(id);
         }
+
+        /// <summary>
+        /// the values of the types contained in the inheriting class
```

Comment:
I'd omit "values of the ", like this:

```suggestion
        /// the types contained in the inheriting class
```